modules/virt-creating-a-primary-cluster-udn.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_networking/virt-connecting-vm-to-primary-udn.adoc
+
+:_mod-docs-content-type: PROCEDURE                                
+[id="virt-creating-a-primary-cluster-udn_{context}"]                                  
+= Creating a primary cluster-scoped user-defined network
+
+You can connect multiple namespaces to the same primary user-defined network (UDN) to achieve native tenant isolation.
+
+.Prerequisites
+. You have access to the cluster as a user with `cluster-admin` privileges.
+. You have installed the OpenShift CLI (`oc`).
+. You have created a namespace and applied the `k8s.ovn.org/primary-user-defined-network` label.
```

Comment:
Again - I'm not even sure it is true, but if it is, then this comment should be modified in a way that it clarifies, that all the namespaces that are targeted for UDN must be labeled.

Reply:
Yeah, I kind of agree ... 

Our wording should make it clear that the event of labeling the namespace upon creation appropriately, and creating a primary UDN CR in said namespace are what overrides the default cluster network, and attaches teh primary UDN instead. 

... it is implied sure, but, maybe, it's better to be crystal clear here.